workflow/controller/controller.go

```diff
@@ -320,6 +320,18 @@ func (wfc *WorkflowController) startLeading(ctx context.Context, logCtx *log.Ent
 	for i := 0; i < podWorkers; i++ {
 		go wait.Until(wfc.podWorker, time.Second, ctx.Done())
 	}
+	if v, found := os.LookupEnv("CACHE_GC_PERIOD"); found {
+		if v != "false" {
+			cacheGCPeriod, err := time.ParseDuration(v)
```

Comment:
I'm sure there is some `GetEnvDuration(value, defaultValue` func you can use here